client/lib/promote-post/index.ts

```diff
@@ -133,6 +134,11 @@ export enum PromoteWidgetStatus {
 	DISABLED = 'disabled',
 }
 
+export enum BlazeCreditStatus {
+	ENABLED = 'enabled',
+	DISABLED = 'disabled',
```

Comment:
I didn't think we needed a fetching state here, we never need to show a loading indicator to the user, because not all users are eligible